build_steps.py

```diff
@@ -459,6 +505,21 @@ def generate_installer(platform):
         )
     ]
 
+@inject_paths
```

Comment:
I copied this from https://gist.github.com/Araq/d71612cabcbcd3be89a1, but that's probably from before nimsuggest was split out. I'll remove this step for now.